src/components/tab/tab.jsx

```diff
@@ -48,7 +48,7 @@ const Tab = props => {
         ...componentProps
     } = props;
     return React.createElement(element, {
-        className: classNames(styles.tab, className),
+        className: classNames(eval("styles.tab" + ((props.selected == true) ? "Selected" : "")), className),
```

Comment:
Eval is generally a bad idea. I'm actually not sure why eval is even needed here?

You could do something like this:

```js
const style = (props.selected == true) ? styles.tabSelected : styles.tab

// or this, if you wanted (it's more similar to the way you used eval)
const style = styles['tab' + (props.selected == true ? 'Selected' : '')]
// but I'd go for the first.
```